CHANGELOG.md

```diff
@@ -2,12 +2,10 @@
 
 ## 0.0.11
 
-### Breaking change & Upgrade Instruction
-- **BREAKING CHANGE - save config before update**: Now supports **list** of VINS and MAC addresses. You will need to adjust configuration. Existing **entities** from v0.0.10f will not be affected.
-- Cut & Paste your current vin to vin_list
-- Cut & Paste your current mac_addr to mac_addr_list
-
 ### Changed
+
+**WARNING WARNING - DO NOT UPGRADE PRIOR TO READ THE BELOW UPGRADE INSTRUCTIONS, SEE AFTER LIST OF CHANGES**
```

Comment:
Can we put this in red?

Reply:
If we can do html code like <p> <br> , perhaps we can put that in red... 🤔 

Yes we can but ... the result is far to be perfect. This is the best that I could do:

<img width="483" alt="2024-07-12_10-04-44" src="https://github.com/user-attachments/assets/b90b206d-fbd4-4eb8-b497-a76f4d5f23b2">

Putting the 2 warning on the same line, the space is just very odd... there's no space between the 2 words and regardless on same line or not, the W and A have different width than the other letters.

Reply:
Woot ; I was able to find the proper way to keep the original font 🎉